cves/CVE-2014-8541.yml

```diff
@@ -268,8 +294,12 @@ i18n:
 
     Answer should be boolean. Write a note about how you came to the conclusions
     you did.
-  answer:
-  note:
+  answer: false
+  note: |
+    A little torn on this option, while the various types (and implementations)
+    of codecs is itself an internationalization issue, I would not say that this specific
+    issue arises from internationalization concerns (of relation to character sets, text
+    encoding, etc).
```

Comment:
Yeah I would agree that it's not. Might be more likely to be found by international files, but it's not an i18n issue specifically